### PR TITLE
Provide more context for Example 1

### PR DESCRIPTION
**Pull request by wardpenney**
* Focus on the topic of the default Neat Grid

screen shot 2017-03-17 at 2 45 34 pm

### CODE REVIEW

File: source/examples/index.html.slim [hunk 1]
```diff
@@ -15,15 +15,21 @@ header.hero.container
 
 section.container.container--spacing
   .column
+    h2.h5
+      | Using the Default Neat Grid
+    p
+      ' This `.default-neat-grid` pattern calls the `grid-container` mixin
```
Comment (whmii) [inline source/examples/index.html.slim]: I think you can add `markdown:` to this and it will parse it

Comment (whmii) [inline source/examples/index.html.slim]: (or something, check the home page)

File: source/examples/index.html.slim [hunk 2]
```diff
@@ -15,15 +15,21 @@ header.hero.container
...
+      ' without passing a custom a grid. It therefore gets the default
```
Comment (whmii) [inline source/examples/index.html.slim]: Introducing "custom a grid" is probably not necessary here

File: source/examples/index.html.slim (hunk 1 above)
Comment (wardpenney) [inline source/examples/index.html.slim]: 🎉

File: source/examples/index.html.slim [hunk 3]
```diff
@@ -15,15 +15,21 @@ header.hero.container
...
+      ' grid that comes with Neat (12-column, 20px gutters, and no max-width).
```
Comment (whmii) [inline source/examples/index.html.slim]: > grid that comes with Neat which is 12 columns and 20px wide gutters

☝️ this way it doesn't look like code

**Comment by wardpenney** [comment]
screen shot 2017-03-17 at 3 05 28 pm

**Comment by wardpenney** [comment]
screen shot 2017-03-17 at 3 17 31 pm

**Pull request merged**